openlibrary/plugins/openlibrary/js/realtime_account_validation.js

```diff
@@ -14,6 +16,9 @@ export function initRealTimeValidation() {
         // Checks whether reportValidity exists for cross-browser compatibility
         // Includes invalid input count to account for checks not covered by reportValidity
         function submitCreateAccountForm() {
+            const submitBtn = $('button[name=signup]');
+            submitBtn.prop('disabled', true).text(i18nStrings['loading_text']);
```

Comment:
Oh if there's an error, the text remains as "Loading..."; is there a way we can prevent that?

Reply:
Hmm good question! I will investigate.